src/libraries/System.Private.Xml/src/System/Xml/Xsl/XsltOld/Compiler.cs

```diff
@@ -1184,7 +1184,7 @@ public ErrorXPathExpression(string expression, string baseUri, int lineNumber, i
             public override XPathExpression Clone() { return this; }
             public override void CheckErrors()
             {
-                throw new XsltException(SR.Xslt_InvalidXPath, new string[] { Expression }, _baseUri, _linePosition, _lineNumber, null);
+                throw new XsltException(SR.Xslt_InvalidXPath, new string[] { Expression }, _baseUri, _lineNumber, _linePosition, null);
```

Comment:
Considering this only affects message suffix (`An error occurred at {0}, ({1}, {2}).`) it doesn't make much difference and hopefully people don't rely on that message and use properties instead. I'm wondering if for consistency with previous versions we should consider also swapping order when displaying: https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.Xml/src/System/Xml/Xslt/XsltException.cs#L126

I don't feel strongly either way

Reply:
> I'm wondering if for consistency with previous versions we should consider also swapping order when creating message

I'm not following.  Can you elaborate?

Reply:
with this change I assume we will swap the `(col, row)` in the error message with `(row, col)` for XSLT, it will be more consistent with rest of the XML but it will produce different message than on older versions.

Reply:
In all error/warning messages I am familiar with across various platforms, (row, col) seems to be the standard. It's also assumed by various tools such as VS. So it's the right fix IMO.

Reply:
> with this change I assume we will swap the `(col, row)` in the error message with `(row, col)` for XSLT, it will be more consistent with rest of the XML but it will produce different message than on older versions.

no, pr is simply fixing 4th and 5th arguments which were wrong before:
https://github.com/dotnet/runtime/blob/08aff106db7ddbb58cc341ce6a178cec4cd516ef/src/libraries/System.Private.Xml/src/System/Xml/Xslt/XsltException.cs#L86